corehq/apps/zapier/models.py

```diff
@@ -0,0 +1,14 @@
+import requests
+from django.db import models
+
+
+class Subscription(models.Model):
```

Comment:
maybe we could rename this `ZapierSubscription` to not be confused with the same model in accounting?